packages/astro/test/dev-routing.test.js

```diff
@@ -247,4 +247,52 @@ describe('Development Routing', () => {
 			expect(body.title).to.equal('data [slug]');
 		});
 	});
+
+	describe('file format routing', () => {
+		/** @type {import('./test-utils').Fixture} */
+		let fixture;
+		/** @type {import('./test-utils').DevServer} */
+		let devServer;
+
+		before(async () => {
+			fixture = await loadFixture({
+				build: {
+					format: 'file',
+				},
+				root: './fixtures/without-site-config/',
+				site: 'http://example.com/',
+			});
+			devServer = await fixture.startDevServer();
+		});
+
+		it('200 when loading /index.html', async () => {
+			const response = await fixture.fetch('/index.html');
+			expect(response.status).to.equal(200);
+		});
+
+		it('200 when loading /', async () => {
```

Comment:
👍 glad to see this test, we should still support not including index.html

Reply:
I could have also named this PR "Fun with Regexes" 😄